.github/workflows/pull-requests.yml

```
@@ -196,6 +196,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        shard: [1, 2, 3, 4]
```

Good to use the same list formatting in our workflows. Also, worth keeping shard configuration in one place rather than shard number being up here and maximum number of shards being on the run line?

```suggestion
        shard: 
          - 1/4
          - 2/4
          - 3/4
          - 4/4
```